test/models/announcement_test.exs

```diff
@@ -75,15 +75,34 @@ defmodule Constable.AnnouncementTest do
     test "interests are sorted alphabetically" do
       interest_a = insert(:interest, name: "a")
       interest_b = insert(:interest, name: "b")
+
       insert(:announcement)
-        |> tag_with_interest(interest_b)
-        |> tag_with_interest(interest_a)
+      |> tag_with_interest(interest_b)
+      |> tag_with_interest(interest_a)
+
+      announcement =
+        Announcement.with_announcement_list_assocs()
+        |> Repo.one()
+
+      assert announcement.interests |> List.first() == interest_a
+      assert announcement.interests |> List.last() == interest_b
+    end
+  end
+
+  describe ".for_user/1" do
```

Comment:
This is the new test in this file. The rest of the changes were done by the auto-formatter.